db/sqlc/queries/accounts.sql

```diff
@@ -9,6 +9,15 @@ SET current_balance_msat = $1
 WHERE id = $2
 RETURNING id;
 
+-- name: AdjustAccountBalance :one
+UPDATE accounts
+SET current_balance_msat = current_balance_msat + CASE
+    WHEN sqlc.arg(is_addition)::BOOLEAN THEN sqlc.arg(amount)  -- If IsAddition is true, add the amount
+    ELSE -sqlc.arg(amount)         -- If IsAddition is false, subtract the amount
+END
+WHERE id = sqlc.arg(id)
+RETURNING id;
+
```

Comment:
Let me know if you prefer that I use the current functionality instead to fetch the current balance, and then set the balance to the `currentBalance +- amount`.
My motivation for introducing this was to discourage such functionality in the future, which directly sets the balance.

Reply:
agreed re having specific queries for how we prefer to use it 👍 
see comment re splitting this out though